src/commands/deploy/latest.js

```diff
@@ -590,7 +589,14 @@ export default async function main(
         }
       }
     } else {
-      const deploymentResponse = await now.fetch(deploymentUrl);
+      const deploymentResponse = handleCertError(
+        output,
+        await getDeploymentContentsByUrl(now, deploymentUrl)
```

Comment:
Not using `getDeploymentByIdOrHost` here because it is using `v5` while this code is using `v9` endpoint.